src/UsersView/usermodel.ts

```diff
@@ -0,0 +1,14 @@
+
+
+export class UserModel {
```

Comment:
I'm not quite sure about this class, maybe need Adi to advice

Reply:
you can just create interface instead of a class

```
interface UserModel {
  name: string
  colour: string
}

//then you can
const person: UserModel = {
  name: 'Bruno',
  colour: 'bla'
} 
```

Reply:
Also I am not sure if you need pressStatus